src/main/java/org/generationcp/middleware/api/brapi/v2/observationunit/ObservationUnitServiceImpl.java

```diff
@@ -1,30 +1,81 @@
 package org.generationcp.middleware.api.brapi.v2.observationunit;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
Please address the SonarLint errors in this class.

Reply:
<img width="858" alt="Screen Shot 2021-07-09 at 8 00 06 AM" src="https://user-images.githubusercontent.com/5011207/125004264-c1b6f100-e08b-11eb-9d43-eb967006927b.png">